### PR TITLE
#63/DEV/PageNotFound

### PR DESCRIPTION
**Pull request by n0macx**
- Added Route paths with "*" to all undescribed pages
- - Added NotFoundPage.js and styles files

### CODE REVIEW

File: src/Pages/NotFoundPage/NotFoundPageStyled.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const NotFoundPage = () => {
```
Comment (MonikaKrella) [inline src/Pages/NotFoundPage/NotFoundPageStyled.js:1]: shouldn't be this content in NotFoundPage.js instead of NotFoundPageStyled?

**Review by MonikaKrella — COMMENTED** [review]
-make this small change in files (for which I made a comment) and for me after that it would be approved

File: src/Pages/NotFoundPage/NotFoundPageStyled.js (hunk 1 above)
Comment (n0macx) [reply src/Pages/NotFoundPage/NotFoundPageStyled.js:1]: Fixed

**Review by Smyku6 — APPROVED** [review]
Good job

**Pull request merged**